tests/test_modeling_tf_common.py

```diff
@@ -573,7 +573,12 @@ def check_pt_tf_models(tf_model, pt_model):
             pt_model = pt_model_class(config)
 
             tf_inputs_dict = self._prepare_for_class(inputs_dict, model_class)
-            tf_inputs_dict_maybe_with_labels = self._prepare_for_class(inputs_dict, model_class, return_labels=True)
+            tf_inputs_dict_maybe_with_labels = self._prepare_for_class(
+                inputs_dict,
+                model_class,
+                # Not all models accept "labels" in the forward pass (yet :) )
+                return_labels=True if "labels" in inspect.signature(model_class.call).parameters.keys() else False,
```

Comment:
Some models, like `TFElectraForPreTraining`, do not have a label argument, unlike their PT counterparts. There are 5 instances like this, all `XXXForPretraining` (not to be confused with `XXXPreTrainedModel`, the base models to be inherited).

Without this correction, those models would fail due to the inexisting `label` argument.